sbin/txt-tpm1-evt-log-parser.awk

```diff
@@ -69,9 +71,14 @@ BEGIN {
 		printf("    Digests:\n")
 		printf("      SHA1: ")
 		hexdump($3, 20)
+		sym = "SHA1_" x2n($1, 4)
+		SYMTAB[sym] = SYMTAB[sym] hex_noprint($4, a[2]) "\n"
```

Comment:
```suggestion
		SYMTAB[sym] = SYMTAB[sym] hex_noprint($3, 20) "\n"
```
You must have tested it on TPM2.

Reply:
Right, copy-pasted and forgot to check it. Fixed in https://github.com/TrenchBoot/qubes-antievilmaid/compare/a1196c77efb591cee69ddcce4df20c384a6ffca9..2cda9788812bcdb2531f3314f705a916997c71ae